org.code.toboggan.network/src/org/code/toboggan/network/notification/clientcorelisteners/file/FileMoveNotificationHandler.java

```diff
@@ -27,13 +28,16 @@
 	
 	@Override
 	public void handleNotification(Notification notification) {
-		FileMoveNotification n = (FileMoveNotification) notification.getData();
+		FileMoveNotification n = (FileMoveNotification) notification.getData(); 
 		logger.info("Received file move notification for " + notification.getResourceID());
 		Set<ICoreExtension> extensions = extMgr.getExtensions(NetworkExtensionIDs.FILE_MOVE_NOTIFICATION_ID, IFileMoveNotificationExtension.class);
+		 
+		File file = storage.getFile(notification.getResourceID());
+		Path newAbsolutePath = CoreActivator.getSessionStorage().getProjectLocation(file.getProjectID()).resolve(n.newPath).resolve(file.getFilename());
```

Comment:
Is NetworkUtils still being used at all after changing this? Was wondering if this change should be reflected in the other  toAbsolutePathRelative() calls. If so, it might be worth it to just make the change in that class.